src/game_api/rpc.cpp

```diff
@@ -1130,13 +1130,13 @@ void extinguish_particles(ParticleEmitterInfo* particle_emitter)
         static free_particleemitter_func* fpf = (free_particleemitter_func*)(offset);
         if (particle_emitter != nullptr)
         {
-            if (particle_emitter->unknown26 != 0)

```

Comment:
I didn't actually test the changes I made to this file. I just made it compile in a way that looks like it should be correct.

Reply:
Tested and checked and could actually simplify this